docs/en/sql-reference/sql-statements/loading_unloading/INSERT.md

```diff
@@ -128,3 +151,52 @@ INSERT INTO insert_wiki_edit
         "aws.s3.region" = "ap-southeast-1"
 );
 ```
+
+### Example 3: INSERT timeout
+
+The following example inserts the data from the source table `source_wiki_edit` to the target table `insert_wiki_edit` with the timeout duration set to `2` seconds.
```

Comment:
If you want to ingest large data, you can set a larger value for the `timeout`, or for session variable `insert_timeout`, the default is `insert_load_default_timeout_second`.
（but, this needs to be checked)

Reply:
done